examples/resources/kentik-synthetics_test/resource.tf

```diff
@@ -201,3 +246,129 @@ resource "kentik-synthetics_test" "minimal-url-test" {
     rollup_level = 1
   }
 }
+
+resource "kentik-synthetics_test" "minimal-network-grid-test" {
+  name   = "minimal-network-grid-test"
+  type   = "network_grid"
+  status = "TEST_STATUS_PAUSED"
+  settings {
+    network_grid {
+      targets = ["127.0.0.1"]
+    }
+    agent_ids = [
+      "817",
+      "818",
+      "819"
+    ]
+    tasks = [
+      "ping",
+      "traceroute"
+    ]
+    ping {
+      period = 60
+    }
+    trace {
+      period   = 60
+      protocol = "udp"
+    }
+    port         = 443
+    protocol     = "tcp"
+    family       = "IP_FAMILY_V4"
+    rollup_level = 1
+  }
+}
+
+resource "kentik-synthetics_test" "minimal-page-load-test" {
+  name   = "minimal-page-load-test"
+  type   = "page_load"
+  status = "TEST_STATUS_PAUSED"
+  settings {
+    page_load {
+      target = "https://example.test"
+    }
+    agent_ids = [
+      "817",
+      "818",
+      "819"
+    ]
+    period = 60
+    expiry = 5000
+    tasks = [
+      "ping",
```

Comment:
Must include `page-load` otherwise the test is equivalent to `hostname` test.

Reply:
Done